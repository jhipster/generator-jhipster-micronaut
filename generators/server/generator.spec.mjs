import expect from 'expect';

import { helpers, lookups } from '#test-utils';

describe('SubGenerator server of micronaut JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create('jhipster:server')
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'micronaut',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
