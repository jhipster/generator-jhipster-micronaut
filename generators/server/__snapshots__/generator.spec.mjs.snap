// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator server of micronaut JHipster blueprint run should succeed 1`] = `
Object {
  ".editorconfig": Object {
    "stateCleared": "modified",
  },
  ".gitattributes": Object {
    "stateCleared": "modified",
  },
  ".gitignore": Object {
    "stateCleared": "modified",
  },
  ".husky/pre-commit": Object {
    "stateCleared": "modified",
  },
  ".lintstagedrc.js": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/MavenWrapperDownloader.java": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/maven-wrapper.jar": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/maven-wrapper.properties": Object {
    "stateCleared": "modified",
  },
  ".prettierignore": Object {
    "stateCleared": "modified",
  },
  ".prettierrc": Object {
    "stateCleared": "modified",
  },
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "README.md": Object {
    "stateCleared": "modified",
  },
  "README.md.jhi": Object {
    "state": "modified",
  },
  "checkstyle.xml": Object {
    "stateCleared": "modified",
  },
  "mvnw": Object {
    "stateCleared": "modified",
  },
  "mvnw.cmd": Object {
    "stateCleared": "modified",
  },
  "npmw": Object {
    "stateCleared": "modified",
  },
  "npmw.cmd": Object {
    "stateCleared": "modified",
  },
  "openapi.properties": Object {
    "stateCleared": "modified",
  },
  "package.json": Object {
    "stateCleared": "modified",
  },
  "pom.xml": Object {
    "stateCleared": "modified",
  },
  "sonar-project.properties": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/app.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/grafana/provisioning/dashboards/JVM.json": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/grafana/provisioning/dashboards/dashboard.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/grafana/provisioning/datasources/datasource.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/jib/entrypoint.sh": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/monitoring.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/prometheus/prometheus.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/docker/sonar.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/JhipsterApp.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/ActiveProfilesInfoSource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/ApplicationProperties.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/Constants.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/DefaultProfileUtil.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/JHipsterConfigurationEndpoint.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/JacksonConfiguration.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/LoggingConfiguration.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/MessagesBundleMessageSource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/SnakeCasePhysicalNamingStrategy.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/metric/JHipsterMetricsEndpoint.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/domain/Authority.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/domain/User.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/domain/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/repository/AuthorityRepository.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/repository/UserRepository.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/repository/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/AuthoritiesConstants.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/BcryptPasswordEncoder.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/DatabaseAuthenticationProvider.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/Logout.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/NotAuthenticatedResponse.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/PasswordEncoder.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/SecurityHeaderFilter.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/SecurityUtils.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/UserNotActivatedException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/security/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/MailSenderFactory.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/MailService.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/UserService.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/dto/AdminUserDTO.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/dto/PasswordChangeDTO.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/dto/UserDTO.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/dto/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/mapper/UserMapper.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/mapper/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/service/util/RandomUtil.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/util/HeaderUtil.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/util/JHipsterProperties.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/util/PaginationUtil.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/AccountResource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/ClientForwardController.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/PublicUserResource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/SwaggerResource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/UserResource.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/BadRequestAlertException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/EmailAlreadyUsedException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/EmailNotFoundException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/ErrorConstants.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/FieldErrorVM.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/InvalidPasswordException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/LoginAlreadyUsedException.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/handlers/AbstractThrowableProblemHandler.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/handlers/ConstraintViolationExceptionHandler.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/handlers/ProblemHandler.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/handlers/ProblemRejectionHandler.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/handlers/UnsatisfiedRouteExceptionHandler.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/errors/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/vm/KeyAndPasswordVM.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/vm/LoginVM.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/vm/ManagedUserVM.java": Object {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/vm/package-info.java": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/application-dev.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/application-tls.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/application.yml": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/i18n/messages.properties": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/i18n/messages_en.properties": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/logback.xml": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/views/error.html": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/views/mail/activationEmail.html": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/views/mail/creationEmail.html": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/views/mail/passwordResetEmail.html": Object {
    "stateCleared": "modified",
  },
  "src/main/resources/views/mail/testEmail.html": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/activate.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/configuration.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/error.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/global.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/health.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/home.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/login.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/logs.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/metrics.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/password.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/register.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/reset.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/sessions.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/settings.json": Object {
    "stateCleared": "modified",
  },
  "src/main/webapp/i18n/en/user-management.json": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/config/CorsController.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/config/CorsTest.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/config/timezone/HibernateTimeZoneIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/repository/timezone/DateTimeWrapper.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/repository/timezone/DateTimeWrapperRepository.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/security/DomainUserDetailsServiceIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/security/SecurityHeaderFilterTest.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/security/SecurityUtilsUnitTest.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/security/jwt/JWTFilterTest.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/service/MailServiceIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/service/UserServiceIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/service/mapper/UserMapperIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/AccountResourceIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/ClientForwardControllerIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/TestUtil.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/UserJWTControllerIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/UserResourceIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/errors/ExceptionTranslatorIT.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/errors/ExceptionTranslatorTestController.java": Object {
    "stateCleared": "modified",
  },
  "src/test/java/com/mycompany/myapp/web/rest/errors/TestDTO.java": Object {
    "stateCleared": "modified",
  },
  "src/test/resources/application-test.yml": Object {
    "stateCleared": "modified",
  },
  "src/test/resources/i18n/messages_en.properties": Object {
    "stateCleared": "modified",
  },
}
`;
